hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/OMLayoutFeature.java

```diff
@@ -32,7 +32,10 @@ public enum OMLayoutFeature implements LayoutFeature {
   INITIAL_VERSION(0, "Initial Layout Version"),
 
   ERASURE_CODED_STORAGE_SUPPORT(1, "Ozone version with built in support for"
-      + " Erasure Coded block data storage.");
+      + " Erasure Coded block data storage."),
+
+  BUCKET_LAYOUT_SUPPORT(2, "Ozone version with built in support for"
```

Comment:
In the future, we might have even more types of bucket layouts added, which will need their own layout features. Since this version is introducing the concept of bucket layouts as well as the OBS and FSO layout types, I think we should update the description string to clarify that these are the layout types brought in with this feature, something like this:
`"Ozone version supporting bucket layouts and introducing the FSO and OBS bucket layout types."`

Reply:
Thank you for the suggestion Ethan, I've modified the description.